app/src/main/java/com/dragic/gamehunter/view/uicomponents/DealCard.kt

```diff
@@ -114,7 +115,11 @@ fun DealCard(
                                 shape = RectangleShape
                             )
                             .padding(horizontal = dimensionResource(id = R.dimen.deal_card_save_percentage_padding)),
-                        text = savePercentage,
+                        text = buildAnnotatedString {
+                            append("-")
+                            append(savePercentage)
+                            append(stringResource(id = R.string.savings_off))
+                        },
```

Comment:
Building the string like this is a bit confusing and difficult to localize. Could you make it a parametrized string resource? Something like this

```xml
<string name="savings_template">-%s OFF</>
```